hw_i2c/sample-implementations/linux_user_space/sensirion_hw_i2c_implementation.c

```diff
@@ -67,6 +67,14 @@ void sensirion_i2c_init(void) {
         return; /* no error handling */
 }
 
+/**
+ * Release all resources initialized by sensirion_i2c_init().
+ */
+void sensirion_i2c_release(void) {
+    if (i2c_device >= 0)
```

Comment:
Is 0 really a valid i2c_device? Isn't file descriptor 0 always stdin?

Reply:
Thinking about it: One may want to simulate I2C over stdin/stdout, so it may be a valid use case.

Reply:
> The  return  value of open() is a file descriptor, a small, nonnegative integer that is used in subsequent  system  calls

`man 2 open`